src/main/java/com/education/repository/TimetableRepository.java

```diff
@@ -3,38 +3,37 @@
 import com.education.domain.Student;
 import com.education.domain.Timetable;
 import com.education.domain.TimetableStatus;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 import java.time.LocalDate;
 import java.time.LocalTime;
 import java.util.List;
 
-public interface TimetableRepository {
-    void save(Timetable timeTable);
+public interface TimetableRepository extends JpaRepository<Timetable, Long> {
 
-    Timetable getById(long id);
+    List<Timetable> findByTeacherId(long teacherId);
```

Comment:
Why List?

Reply:
because it finds timetables with TeacherId
I think I should write the name of the method completely